### DISCUSSION TITLE
Flag for hidden directory.

### DISCUSSION THREAD

**Discussion by itzcodex24** [junegunn/fzf#3418] [Q&A]
Hi there, is the a way I can use the fzf command within the terminal to search for files excluding those in a directory specified.

Something like this is what I'm looking for:
`fzf --hide node_modules/*`

**Comment by LangLangBart** [comment]
- the feature would probably be rejected by the project's maintainer, since fzf's job is not to curate the list; this should be done by a preprocessor
  - refs:
    - https://github.com/junegunn/fzf/issues/1763#issuecomment-555278293 
    - https://github.com/junegunn/fzf/issues/2095#issuecomment-653849942  
    - ... more [junegunn/fzf/issues labeled wontfix](https://github.com/junegunn/fzf/issues?q=label%3Awontfix+is%3Aissue) 

---

> Without STDIN pipe, fzf will use find command to fetch the list of files excluding hidden ones. (You can override the default command with FZF_DEFAULT_COMMAND) Source: [README.md#usage](https://github.com/junegunn/fzf/blob/master/README.md#usage)

- If you want to exclude the `node_modules` folder you need to set up the `FZF_DEFAULT_COMMAND`, see the following post for more details: https://github.com/junegunn/fzf/issues/1625#issuecomment-507674106

---

- my setup for  `FZF_DEFAULT_COMMAND` using [sharkdp/fd](https://github.com/sharkdp/fd) to ignore `node_modules`

```sh
# .zshrc
export FZF_DEFAULT_COMMAND="fd --hidden --follow --exclude '.{cache,DS_Store,gem,git,npm,parallel,Trash,vscode-oss}' --exclude '{Library,Music,node_modules,Pictures}/' --color always"
